<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>SpeedCommandWidget</class>
	<widget class="QWidget" name="SpeedCommandWidget">
		<layout class="QVBoxLayout" name="mainLayout">
			<item>
				<widget class="QPushButton" name="returnBackBtn">
					<property name="text">
						<string>Back</string>
					</property>
				</widget>
			</item>
			<item>
				<layout class="QHBoxLayout" name="contentLayout">
					<item>
						<widget class="QLabel" name="infoLabel">
							<property name="text">
								<string>Frame delay:</string>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QSpinBox" name="frameDelayInput">
							<property name="minimum">
								<number>20</number>
							</property>
							<property name="maximum">
								<number>1000</number>
							</property>
						</widget>
					</item>
					<item>
						<spacer name="hSpacer">
							<property name="orientation">
								<enum>Qt::Horizontal</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>40</width>
									<height>20</height>
								</size>
							</property>
						</spacer>
					</item>
					<item>
						<widget class="QPushButton" name="applyButton">
							<property name="text">
								<string>Apply</string>
							</property>
						</widget>
					</item>
				</layout>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
